website/versioned_docs/version-0.19.0/forms.md

```diff
@@ -93,7 +93,7 @@ shake the textinput if not a falsy value and different from the previous value
 | shake     | shake the textinput, eg `this.refs.someInputRef.shake()` |
 | focus     | call focus on the textinput ([example](#calling))        |
 | blur      | call blur on the textinput ([example](#calling))         |
-| clearText | call clear on the textinput ([example](#calling))        |
+| clear     | call clear on the textinput ([example](#calling))        |
```

Comment:
Don't change this. The method in 0.19.0 was actually clearText https://github.com/react-native-training/react-native-elements/blob/v0.19.1/src/form/FormInput.js#L60